rspace/src/main/scala/coop/rchain/rspace/ISpace.scala

```diff
@@ -5,7 +5,13 @@ import coop.rchain.rspace.internal._
 
 import scala.collection.immutable.Seq
 
-case class Result[R](value: R, persistent: Boolean)
+final case class Result[R](value: R, persistent: Boolean)
```

Comment:
Looks like this isn't used anywhere anymore. In that case why don't you remove this and rename `ContResult` to `Result`?